### PR TITLE
feat(ui5-title): add new "size" property

### PR DESCRIPTION
**Pull request by TeodorTaushanov**
The default size of the title is H5. Components, which use ui5-title are adapted.

### CODE REVIEW

File: packages/main/src/Title.ts [hunk 1]
```diff
@@ -50,12 +50,23 @@ class Title extends UI5Element {
...
+	 * This property does not influence the style of the control.
+	 * Use the property `size` for this purpose instead.
 	 * @default "H2"
 	 * @public
 	 */
 	@property()
 	level: `${TitleLevel}` = "H2";
 
+	/**
+	 * Defines the style of the title.
```
Comment (dimovpetar) [inline packages/main/src/Title.ts:62]: My suggestion is `Defines the visual appearance of the title.`

File: packages/main/test/pages/Title.html [hunk 2]
```diff
@@ -25,12 +25,20 @@
...
 		<ui5-title id="titleH6" level="H6">Title Level 6</ui5-title>
 		<ui5-title level="H2">A Very Long Long Long Long Title Text with Level Two</ui5-title>
+		<br>
+		<ui5-title size="H1">Title Style 1</ui5-title>
+		<ui5-title size="H2">Title Style 2</ui5-title>
+		<ui5-title size="H3">Title Style 3</ui5-title>
+		<ui5-title size="H4">Title Style 4</ui5-title>
+		<ui5-title>Title Style Auto (5)</ui5-title>
+		<ui5-title size="H5">Title Style 5</ui5-title>
+		<ui5-title size="H6">Title Style 6</ui5-title>
```
Comment (dimovpetar) [inline packages/main/test/pages/Title.html:35]: minor: `Title Size 1/2/3...`

File: packages/main/src/themes/PopupsCommon.css [hunk 3]
```diff
@@ -43,8 +43,6 @@
 .ui5-popup-footer-root,
 :host([header-text]) .ui5-popup-header-text {
     margin: 0;
-    font-size: 1rem;
-    font-family: "72override", var(--_ui5_popup_header_font_family);
```
Comment (dimovpetar) [inline packages/main/src/themes/PopupsCommon.css:47]: Looks like this is needed. Dialog's title now appears in `Times New Roman` with 1.5rem font size

**Review by dimovpetar — CHANGES_REQUESTED** [review]

File: packages/main/src/themes/PopupsCommon.css (hunk 3 above)
Comment (TeodorTaushanov) [reply packages/main/src/themes/PopupsCommon.css:47]: ui5-dialog and ui5-popover templates are adapted as well

**Review by ilhan007 — APPROVED** [review]

**Review by tsanislavgatev — APPROVED** [review]
Our components look good to me.

File: packages/main/src/Dialog.hbs [hunk 4]
```diff
@@ -20,7 +20,7 @@
 				{{#if header.length }}
 					<slot name="header"></slot>
 				{{else}}
-					<h1 id="ui5-popup-header-text" class="ui5-popup-header-text">{{headerText}}</h1>
+					<ui5-title level="H1" id="ui5-popup-header-text" class="ui5-popup-header-text">{{headerText}}</ui5-title>
```
Comment (nnaydenow) [inline packages/main/src/Dialog.hbs:23]: you have to describe `ui5-title` as dependency of `ui5-dialog`

**Pull request merged**